webviz_subsurface/containers/_inplace_volumes_onebyone.py

```diff
@@ -285,31 +271,33 @@ def ensemble_selector(self):
         """Dropdown to select ensemble"""
         return html.Div(
             style={"paddingBottom": "30px"},
-            children=[
-                html.Label("Ensemble"),
-                dcc.Dropdown(
-                    id=self.ids("ensemble"),
-                    options=[
-                        {"label": i, "value": i}
-                        for i in list(self.volumes["ENSEMBLE"].unique())
-                    ],
-                    clearable=False,
-                    value=list(self.volumes["ENSEMBLE"])[0],
-                ),
-            ],
+            children=html.Label(
+                children=[
+                    html.Span("Ensemble:", style={"font-weight": "bold"}),
+                    dcc.Dropdown(
+                        id=self.ids("ensemble"),
+                        options=[
+                            {"label": i, "value": i}
+                            for i in list(self.volumes["ENSEMBLE"].unique())
+                        ],
+                        clearable=False,
+                        value=list(self.volumes["ENSEMBLE"])[0],
+                    ),
+                ]
+            ),
         )
 
     @property
     def plot_selector(self):
         """Radiobuttons to select plot type"""
         return html.Div(
             children=[
-                html.Label("Plot Type"),
+                html.Span("Plot type:", style={"font-weight": "bold"}),
                 dcc.RadioItems(
                     id=self.ids("plot-type"),
                     options=[
                         {"label": i, "value": i}
-                        for i in ["Per Realization", "Box Plot"]
+                        for i in ["Per realization", "Box plot"]
```

Comment:
If the options are to be changed then the value and the callbacks must be changed as well.